extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -157,7 +157,7 @@ public static boolean createTableIfNotExists(final Configuration conf, final Str
         return false;
     }
 
-    private static String getIndexTableName(final Configuration conf, final String indexTableNameConf, final String altSuffix){
```

Comment:
try to avoid changes like this that don't change logic or intent.  the finals are annoying, but it makes it hard to parse what changed.